lib/ansible/modules/windows/win_certificate_store.py

```diff
@@ -48,6 +48,14 @@
     description:
     - The store name to use when importing a certificate or searching for a
       certificate.
+    - C(AddressBook): The X.509 certificate store for other users
```

Comment:
Because you have `: ` (with the space), you need to quote the entire line for it to be valid yaml.

Another option, instead of adding the description for each option is to just give a link to the Enum in the .net docs https://msdn.microsoft.com/en-us/library/system.security.cryptography.x509certificates.storename.aspx. I'll leave it up to you what you would like to have as I don't mind either way.

Reply:
Yep, ok sorry about that.  I think I'll leave the (mostly won't change) text.  MS have a nice habit of breaking their links.